lib/modules/apostrophe-docs/lib/api.js

```diff
@@ -147,6 +147,13 @@ module.exports = function(self, options) {
   // If `options.permissions` is set explicitly to
   // `false`, permissions checks are bypassed.
   //
+  // If `options.skipAttachments` is true, the insert operation will be slightly
+  // faster, however this is only safe to use if it is impossible for the
+  // document to directly contain any attachments. This includes attachments
```

Comment:
"directly contain"? Do you  mean having an attachment field in the schema of the doc type?